torch/_decomp/decompositions.py

```diff
@@ -853,7 +853,7 @@ def native_group_norm(
     eps: float,
 ) -> Tuple[Tensor, Tensor, Tensor]:
     orig_shape = input.shape
-    input = input.view(N, group, C // group, HxW)
+    input = input.view((N, group, C // group, HxW))
```

Comment:
this shouldn't be necessary anymore, view ref should accept unpacked size?

Reply:
When I wrote this it didn't accept unpacked size, but I believe that has since been fixed. I will undo this change.